.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  sbt:
+    name: Publish to apache nexus
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko-sbt-paradox'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 100
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
+
+      - name: Publish
+        run: |-
+          sbt publish
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
```

Comment:
The environment variables are wrong, see https://github.com/apache/incubator-pekko/blob/main/.github/workflows/publish-nightly.yml#L62-L64

Reply:
Ugh, I thought I copied it from some pekko project but maybe I misremixed...

Reply:
Probably copied from [here](https://github.com/apache/incubator-pekko/blob/main/.github/workflows/publish.yml) which probably hasn't been updated yet.

Reply:
Done